src/main/java/com/azure/cosmos/examples/common/UserSession.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.common;
+
+public class UserSession {
+    public UserSession()
+    {
+
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getTenantId(){return this.tenantId;}
+
+    public void setTenantId(String tenantId){this.tenantId = tenantId;}
+
+    public String getUserId(){return this.userId;}
+
+    public void setUserId(String userId){this.userId = userId;}
+
+    public String getSessionId(){return this.sessionId;}
+
+    public void setSessionId(String sessionId){this.sessionId = sessionId;}
+
+    private String id="";
```

Comment:
As per java convention, we should declare these instance variables on the top of the class. 
Also, why do we need these default "" string values ?